content/post/2020-05-08-ardn-hackathon.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Hackathon to Put TERRA REF on BrAPI
+authors: 
+- Kristina Riemer, David LeBauer
+date: '2020-05-08'
+slug: ardn-hackathon
+categories: []
+tags: []
+subtitle: ''
+summary: ''
+featured: no
+image:
+  caption: ''
+  focal_point: ''
+  preview_only: no
+projects: []
+---
+
+![Zoom Group Photo of Hackathon Attendees](/post/2020-05-08-ardn-hackathon_files/ARDN_team_2020.05.jpg)
+
+## Outline
+
+- paragraph 1
+  - What: four day virtual hackathon for Agricultural Research Data Network (ARDN) project
+  - Our group's participants were David and Kristina
+  - funded by NIFA FACT grant to
+
+- paragraph 2
+  - Define ARDN project: making agricultural datasets for crop modeling more Interoperable and Reusable by converting them into a compatible format (AgMIP/ICASA), making them more Findable and Accessible by putting them on an easy-to-use platform (ADC NAL = USDA repository for agricultural data)
+  - UF folks leading this
+  - also NAL folks are involved
+
+- paragraph 3
+  - Brief summary of other datasets
+  - ISU
+  - MSU
+  - UGA
+
+- paragraph 4
+  - Summary of our part: putting TR data from BrAPI into ARDN, so any BrAPI-compliant datasets can be automated into that format
+
+- paragraph 5
+  - Expanding to even more datasets, through Vmapper for CSV and for EML (DataONE datasets compatible)
```

Comment:
and our vision - there is definitely not a single one-size-fits-all data standard, but when communities coalesce around a few, it is easier to build tools to make these interoperate, and thus harmonize data.